src/__tests__/data/StatelessDisplayNameFolder/Stateless.d.ts

```diff
@@ -0,0 +1,7 @@
+export interface FooProps {
+  foo?: string;
+}
+
+declare const Foo: React.FC<FooProps>;
+
+export default Foo;
```

Comment:
This was intentionally named `Foo` in order to test the file name based logic is working.